plugins/ci/skills/e2e-test-lifecycle/SKILL.md

```
@@ -0,0 +1,1087 @@
+> current-context: fake
+> users: [{name: fake, user: {token: fake}}]
+> EOF
+> KUBECONFIG=/tmp/fake-kubeconfig ./openshift-tests list tests -o names
+> ```
+> So always list with the fake kubeconfig for a complete enumeration.
+> (Note: piping `list` to `head`/`grep -m` closes the pipe early and prints a
+> `FatalErr` goroutine trace on SIGPIPE — that is a broken-pipe artifact, not
+> a kubeconfig error; redirect to a file if it's noisy.) Component OTE
+> binaries built with the framework directly do not have this tree-build
+> dependency and list completely with no kubeconfig at all.
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)